vscode/resources/notebookTemplate.ipynb

```diff
@@ -0,0 +1,237 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "a2f48f2b",
+      "metadata": {},
+      "source": [
+        "## Azure Quantum Q# notebook\n",
+        "\n",
+        "To use Jupyter Notebooks with Q#, the `qsharp-lang` and `jupyterlab` Python packages should be installed.\n",
+        "\n",
+        "To render charts as shown in this notebook, Matplotlib with notebook widgets support should be installed via `ipympl`.\n",
+        "\n",
+        "To submit to Azure Quantum as shown in this notebook, the `azure-quantum` package needs to be installed.\n",
+        "\n",
+        "You can install all the above packages in your Python environment by running the below in your terminal:\n",
+        "\n",
+        "```bash\n",
+        "pip install jupyterlab qsharp-lang ipympl azure-quantum\n",
```

Comment:
I don't know how necessary this is, but this can actually be a code cell that uses the `%pip install` magic so that it can be executed in the notebook. Though... if we install all the packages at once like this, the output ends up being pretty unreadable in the notebook... so maybe the bash script is a better experience after all.

Reply:
Yeah, I didn't want to break them all out into separate pip installs, but didn't want to omit any either. Plus, I forget which but it asks you to restart the kernel afterwards, so figured it was easier to just tell the user that these prereqs should be in the environment (many of them will be if using something like Conda, but wont if using a clean Python install)